<launch>

  	<node 	pkg="bebop2_state_estimator"
    		type="position_velocity_orientation_filter_node"
    		name="position_velocity_orientation_filter_node_1"
    		respawn="false"
    		launch-prefix=""
    		required="true">
		
			<remap from="/bebop/pose" to="/Bebop1/pose"/>
			<remap from="/bebop/position_velocity_orientation_estimation" to="/Bebop1/position_velocity_orientation_estimation"/>
  	</node>

</launch>
